src/App.css

```diff
@@ -42,6 +42,18 @@ ol, ul {
   margin-top: 2em;
   margin-bottom: 2em;
   padding: 2em 2em;
-  border: 2px solid black;
-  font-weight: bold;
-}
\ No newline at end of file
+  border: 1px solid black;
+}
+
```

Comment:
(can't comment on code that hasn't been changed, but) navbar-right should be a class name, since it can be applied to multiple HTML elements. Let's reserve ids for unique elements